dirty_cat/super_vectorizer.py

```diff
@@ -46,6 +45,30 @@ def _replace_missing_in_col(df: pd.Series, value: str = "missing") -> pd.Series:
     return df
 
 
+def _set_handle_unknown(transformer: BaseEstimator, handle_unknown: str) -> BaseEstimator:
+    """
+    Sets the `handle_unknown` parameter of a transformer. Input is in {"error", "ignore"},
+    converts it if for OrdinalEncoder.
+    """
+    # If low_card_cat_transformer is a string, it's drop or passthrough
+    if isinstance(transformer, str):
+        return transformer
+    if handle_unknown == "error":
+        # Either the transformer has default value and we don't need to change it
+        # Or the user has changed the default value and we don't want to overwrite it
+        return transformer
+    elif handle_unknown == "ignore":
+        if isinstance(transformer, OneHotEncoder) or isinstance(transformer, TargetEncoder):
+            return transformer.set_params(handle_unknown="ignore")
+        elif isinstance(transformer, OrdinalEncoder):
+            return transformer.set_params(handle_unknown="use_encoded_value", unknown_value=np.nan)
+        else:
+            warn("handle_unknown parameter cannot be changed with the "
+                 "transformer you chose for low_card_cat_transformer.")
+            return transformer
```

Comment:
I fear that a typo in "handle_unknown" will lead to a silent or incomprehensible error.

Here we need an "else" clause which explicitly raises a error (I would use a "ValueError"). To improve the user experience, the message of the error should indicate the value that was given, and the acceptable values for instance f'{repr(handle_unknown)} was given, while the valid values are "error" or "ignore"')